documentation/modules/exploit/multi/http/motioneye_auth_rce_cve_2025_60787.md

```diff
@@ -0,0 +1,58 @@
+## Vulnerable Application
+
+This module exploits a template injection vulnerability in the MotionEye.
```

Comment:
In the MotionEye client? Frontend? This sentence ends abruptly and MotionEye has several components, so would be good to clarify which component is affected.